CSSS/Bootstrap.cs

```diff
@@ -191,7 +191,7 @@ private void ShowUsage()
 
             // Argument combinations to pass
             Console.WriteLine("Usage:");
-            Console.WriteLine("  CSSS.exe -c | -o | -p | -s | [--shutdown] | [-h] | [-m]");
+            Console.WriteLine("  CSSS.exe -c | -o | -p | -s [--shutdown] | [-h] | [-m]");

```

Comment:
The `[--shutdown]` argument needs to be with the `-p` argument, not the `-s` one.